spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestPartitionValues.java

```diff
@@ -491,6 +491,7 @@ public void testReadPartitionColumn() throws Exception {
             .option(SparkReadOptions.VECTORIZATION_ENABLED, String.valueOf(vectorized))
             .load(baseLocation)
             .select("struct.innerName")
+            .orderBy("struct.innerName")
```

Comment:
Required by the check below as the default distribution changes the order of elements.